test/acceptance/features/claimForApplicationByName.feature

```diff
@@ -181,4 +180,4 @@ Feature: Claim for an Application by Name
             name: stage-app
         """
         Then On Primaza Cluster "main", the status of ServiceClaim "sc-test" is "Resolved"
-        And On Worker Cluster "worker", ServiceBinding "sc-test" on namespace "applications" state will eventually move to "Malformed"
+        And jsonpath ".status.conditions[0].reason" on "servicebindings.primaza.io/sc-test:applications" in cluster worker is ""NoMatchingWorkloads""
```

Comment:
```suggestion
        And jsonpath ".status.conditions[0].reason" on "servicebindings.primaza.io/sc-test:applications" in cluster worker is "NoMatchingWorkloads"
```

Reply:
Ok, did that because was getting an expection on json.Loads in kubernetescli pkg method.